### PR TITLE
Clarify the meaning of `leg`

### LINKED ISSUE #590: Distinction between meaning of `leg` in various miovision tables is unclear
This incorrectly shows substantial wrong-way car travel on Richmond at Bay. 
```sql
SELECT 
    SUM(volume),
    COUNT(*)
FROM miovision_api.volumes_15min
WHERE 
    classification_uid = 1 -- cars
    AND intersection_uid = 28 -- Richmond and Bay
    AND leg = 'W' -- approaching from the west
```
If the approach leg is 'W', then travel is Eastbound (right?), but Richmond only goes westbound. It's more than a couple random deviants. 

The raw `volumes` table correctly has no such data. 

I'm not sure if there are other instances of this problem. i.e. other intersections, directions.

### PR DESCRIPTION
**Pull request by Nate-Wessel**
## What this pull request accomplishes:

- Will try to clarify the meaning of `leg` with regard to miovision turning movements
- 

## Issue(s) this solves:

- fixes #590 

## What, in particular, needs to reviewed:

-

### CODE REVIEW

**Comment by scann0n** [comment]
@radumas I have added two sections to the Miovision documentation:
1) one section has all of those diagrams + text to better explain legs
2) there's a shorter section on the data validation work we've done with the 🦎s
It's ready for your review!

**Comment by radumas** [comment]
That number 2 should've been in the data validation PR

**Comment by scann0n** [comment]
I have deleted the old images and revamped a bit of the surrounding text. It's ready for re-review...

**Comment by scann0n** [comment]
Oh and I also deleted the QC stuff I put in last time because I did that in much more detail through PR #589

File: volumes/miovision/img/intersection.pptx
Comment (radumas) [inline volumes/miovision/img/intersection.pptx:1]: @scann0n in what software did you make your diagram? If PowerPoint can you upload it?

Comment (scann0n) [reply volumes/miovision/img/intersection.pptx:1]: Implemented!

**Review by radumas — APPROVED** [review]

**Pull request merged**